lib/vrl/compiler/src/expression/group.rs

```diff
@@ -26,6 +26,10 @@ impl Expression for Group {
     fn type_def(&self, state: &State) -> TypeDef {
         self.inner.type_def(state)
     }
+
+    fn dump(&self, vm: &mut crate::vm::Vm) -> Result<(), String> {
+        self.inner.dump(vm)
+    }
```

Comment:
Can we add unit tests for all of these, to somehow(?) ensure that whatever the VM produces after this dump is equivalent to what the regular runtime produces?

Reply:
Yes. That is my next job after this.